src/pages/home/index.js

```diff
@@ -95,11 +94,16 @@ const Home = () => {
     formState: { errors },
   } = useForm({ resolver: zodResolver(homeSchema) });
 
+  const {
+    register: editProfileForm,
+    handleSubmit: handleEditProfileForm,
+    formState: { editProfileErrors },
+  } = useForm({ resolver: zodResolver(profileSchema) });
+
   const onSubmit = data => {
     if (targetsList.length <= 9) {
       createTarget({ ...data, ...latLng });
     } else {
-      setTargetsLimit(true);
       window.alert(t('target.limit.alert'));
```

Comment:
This is not necessary but in the future consider changing this with a custom modal. Alerts are not that UX friendly.